src/app/gamification/game-planet/game-planet.component.html

```diff
@@ -26,7 +26,7 @@ <h3 class="description">{{ planet.description }}</h3>
   <div class="monsters-container">
     <h4>Planet's Monsters</h4>
     <ul class="monsters">
-      <li class="monster" *ngFor="let monster of planet.monsters">
+      <li [class]="'monster' + ((monster.owned) ? ' owned' : '')" *ngFor="let monster of planet.monsters">
```

Comment:
Use this attribute for such cases, it is much more readable and easy
https://stackoverflow.com/questions/49545437/using-angular-ngclass-with-boolean-var

Reply:
You learn new shit everyday. Thanks, updated.